tls/s2n_x509_validator.c

```diff
@@ -874,7 +874,13 @@ S2N_RESULT s2n_x509_validator_validate_cert_stapled_ocsp_response(struct s2n_x50
     int status = 0;
     int reason = 0;
 
-    /* sha1 is the only supported OCSP digest */
+    /* SHA-1 is the only supported OCSP digest due to its wide compatibility and established use
+    * in existing systems. Supporting additional hash algorithms would require changes to error 
+    * handling and compatibility checks, which are not currently justified by user demand. For 
```

Comment:
Would it require changes to error handling to support additional hash algorithms? Also, just mentioning why only SHA1 is supported might be sufficient.